### DISCUSSION TITLE
求问一下，怎么让minecraft和mirai的API互相调用

### DISCUSSION THREAD

**Discussion by AstralFlare-owo** [nonebot/discussions#454] [问答]
在写一个插件，现在不知道怎么在minecraft适配器里面使用mirai适配器向指定群发消息，也不知道怎么从mirai向minecraft发消息，有大佬能帮一下么

**Comment by yanyongyu** [comment]
你可以在接收到mc相关事件后调用mirai的api，或者反过来一样

**Reply by AstralFlare-owo** [reply]
我知道是要调mirai api，但是主要是我自己尝试了调用之后会卡住然后过一会ActionFailed 所以找大佬指点一下 比如对应的是哪个API

**Reply by yanyongyu** [reply]
api你可以参考mirai api http相关的文档内容，你可以根据报错内容调试
